lib/bytesutil/bytesutil.go

```diff
@@ -90,3 +90,13 @@ func LimitStringLen(s string, maxLen int) string {
 	n := maxLen/2 - 1
 	return s[:n] + ".." + s[len(s)-n:]
 }
+
+// ToStringOwned converts bytes into string and returns its copy
+func ToStringOwned(b []byte) string {
```

Comment:
@f41gh7 I thought `string(byteSlice)` already does a new allocation and a copy. What's the difference to this new function?

Reply:
It depends. Compiler may use optimizations for this case and do not perform a copy of string. 

 For now, it's possible only with few cases. But later it may be changed.

Reply:
https://github.com/golang/go/blob/master/src/runtime/string.go#L136

Reply:
Replaced it with runtime `string()` conversion.